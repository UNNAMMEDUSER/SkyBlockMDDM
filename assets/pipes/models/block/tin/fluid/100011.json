{"credit": "Dareda", "parent": "pipes:block/root", "textures": {"0": "pipes:block/fluid/tin", "particle": "pipes:block/fluid/tin"}, "elements": [{"name": "Center", "from": [5, 5, 5], "to": [11, 11, 11], "faces": {"north": {"uv": [6, 10, 12, 16], "texture": "#0"}, "east": {"uv": [6, 10, 12, 16], "texture": "#0"}, "south": {"uv": [6, 10, 12, 16], "texture": "#0"}, "west": {"uv": [6, 10, 12, 16], "texture": "#0"}, "up": {"uv": [6, 10, 12, 16], "texture": "#0"}, "down": {"uv": [6, 10, 12, 16], "texture": "#0"}}}, {"name": "Down", "from": [5, 0, 5], "to": [11, 5, 11], "faces": {"north": {"uv": [0, 5, 6, 10], "texture": "#0"}, "east": {"uv": [0, 5, 6, 10], "texture": "#0"}, "south": {"uv": [0, 5, 6, 10], "texture": "#0"}, "west": {"uv": [0, 5, 6, 10], "texture": "#0"}, "down": {"uv": [0, 10, 6, 16], "texture": "#0"}}}, {"name": "Up", "from": [5, 11, 5], "to": [11, 16, 11], "faces": {"north": {"uv": [0, 5, 6, 10], "texture": "#0"}, "east": {"uv": [0, 5, 6, 10], "texture": "#0"}, "south": {"uv": [0, 5, 6, 10], "texture": "#0"}, "west": {"uv": [0, 5, 6, 10], "texture": "#0"}, "up": {"uv": [0, 10, 6, 16], "texture": "#0"}}}, {"name": "South", "from": [5, 5, 11], "to": [11, 11, 16], "faces": {"east": {"uv": [0, 5, 6, 10], "rotation": 90, "texture": "#0"}, "south": {"uv": [0, 10, 6, 16], "texture": "#0"}, "west": {"uv": [0, 5, 6, 10], "rotation": 90, "texture": "#0"}, "up": {"uv": [0, 5, 6, 10], "texture": "#0"}, "down": {"uv": [0, 5, 6, 10], "texture": "#0"}}}]}