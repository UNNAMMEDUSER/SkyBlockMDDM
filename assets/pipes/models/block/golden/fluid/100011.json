{"credit": "Dareda", "parent": "pipes:block/tin/fluid/100011", "textures": {"0": "pipes:block/fluid/golden"}}